grpc/optimistic/server.go

```diff
@@ -74,7 +73,12 @@ func (o *AuctionServiceV1Alpha1) GetBidStream(_ *auctionPb.GetBidStreamRequest,
 				bid := auctionPb.Bid{}
 
 				totalCost := big.NewInt(0)
-				effectiveTip := cmath.BigMin(pendingTx.GasTipCap(), new(big.Int).Sub(pendingTx.GasFeeCap(), optimisticBlock.BaseFee))
+				effectiveTip, err := pendingTx.EffectiveGasTip(optimisticBlock.BaseFee)
+				if err != nil {
+					log.Warn("effective tip is too low", "effectiveTip", effectiveTip.String())
```

Comment:
This might make sense as a debug or a metric counter + debug log. If people send low tips that's on them, but useful to know if it's happening a lot potentially.

Reply:
yeah good point, i ll add a metric. but ideally they shouldn't even arrive at this stage cause we are filtering txs which sends too low gas fee during tx submission.